pkg/kubelet/secret/secret_manager.go

```diff
@@ -93,15 +97,18 @@ type secretStore struct {
 
 	lock  sync.Mutex
 	items map[objectKey]*secretStoreItem
-	ttl   time.Duration
+
+	ttl    time.Duration
+	getTTL GetObjectTTLFunc
 }
 
-func newSecretStore(kubeClient clientset.Interface, clock clock.Clock, ttl time.Duration) *secretStore {
+func newSecretStore(kubeClient clientset.Interface, clock clock.Clock, getTTL GetObjectTTLFunc, ttl time.Duration) *secretStore {
 	return &secretStore{
 		kubeClient: kubeClient,
 		clock:      clock,
 		items:      make(map[objectKey]*secretStoreItem),
 		ttl:        ttl,
```

Comment:
In this case, I don't think we need a `ttl` field anymore. Just define a default constant in this file is enough.